Missing direct dependency

When running the tests I get:

```
Cannot find module 'dom-testing-library' from 'index.js'
```

In the `README.md` it's not written to install this library and I know that other testing libraries like `react-testing-library` have `dom-testing-library` as direct dependency. Should it be added?

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
   "dependencies": {
     "@babel/runtime": "^7.4.3",
     "core-js": "^3.0.1",
-    "dom-testing-library": "^4.0.0",
+    "dom-testing-library": "^4.0.0"
   },
   "eslintConfig": {
     "extends": "./node_modules/kcd-scripts/eslint.js",

--- a/package.json
+++ b/package.json
@@ -10,7 +10,6 @@
   "devDependencies": {
     "babel-jest": "^24.7.1",
     "deasync": "^0.1.14",
-    "dom-testing-library": "^4.0.0",
     "esm": "^3.2.22",
     "jest": "^24.7.1",
     "jest-dom": "^3.1.2",
@@ -42,7 +41,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.3",
-    "core-js": "^3.0.1"
+    "core-js": "^3.0.1",
+    "dom-testing-library": "^4.0.0",
   },
   "eslintConfig": {
     "extends": "./node_modules/kcd-scripts/eslint.js",

